exampleSite/content/blog/_index/index.md

```diff
@@ -0,0 +1,6 @@
++++
+date = "2018-07-06"
+url = "/blog"
```

Comment:
Not sure I like the `url = "/blog"` workaround. From an implementation perspective I see why, but from a user perspective it feels more complex than it should.
Another option would be to go with this options for now and in a separate release/step work on the defaulting.

Basically a section index, would automatically be a list fragment. Leaf bundle content would automatically be a content fragment.

Reply:
Changed in 9ac6dd681a5d1e8c85a0610be1fe0fe2465dad88 to `/blog/latest`

Reply:
Sorry probably could have stated my point better.
I dislike the usage of `url =` as this is something the user have to change, while knowing how Hugo would interpret it. The location was actually correct in my view.
We have `/blog/article/` so the list should be located at `/blog/`. The users just shouldn't need to know that.